jabgui/src/main/java/org/jabref/gui/linkedfile/LinkedFileEditDialog.java

```diff
@@ -86,6 +87,8 @@ private void initialize() {
         link.textProperty().bindBidirectional(viewModel.linkProperty());
         fileType.valueProperty().bindBidirectional(viewModel.selectedExternalFileTypeProperty());
         sourceUrl.textProperty().bindBidirectional(viewModel.sourceUrlProperty());
+
+        Platform.runLater(() -> link.requestFocus());
```

Comment:
@calixtus This is differrent than the stack overflow solution from https://stackoverflow.com/a/70021819/873282 - is it also OK?

Reply:
Reason: lifecycle could be "problematic" - there is no guarantee that intializate is called right before showing the dialog.